clouddriver-alicloud/src/main/java/com/netflix/spinnaker/clouddriver/alicloud/model/AliCloudCluster.java

```diff
@@ -0,0 +1,55 @@
+package com.netflix.spinnaker.clouddriver.alicloud.model;
+
+import com.netflix.spinnaker.clouddriver.model.Cluster;
+import java.util.Set;
+
+/**
+ * @author: luoguan
+ * @create: 2019-06-18 16:35
+ */
+public class AliCloudCluster implements Cluster {
+
+  private String name;
+  private String type;
+  private String accountName;
+  private Set<AliCloudServerGroup> serverGroups;
+  private Set<AliCloudLoadBalancer> loadBalancers;
```

Comment:
Since you're using Lombok, you could use `@Getter` at the class-level and avoid writing / generating all of those getter-methods.

Reply:
Thanks and I will check it.